pkg/curatedpackages/bundle.go

```diff
@@ -119,3 +121,18 @@ func (b *BundleReader) UpgradeBundle(ctx context.Context, controller *packagesv1
 	}
 	return nil
 }
+
+func GetPackageBundleRef(vb releasev1.VersionsBundle) string {
```

Comment:
Just for my own understanding, what is this supposed to do?
Maybe an example of in->out will help me

Reply:
We currently don't include packagebundle manifests in eks-anywhere release to avoid tight coupling between eks-anywhere and packagebundles. What this is doing is it is getting the curated packages controller (which is in the release manifest) uri (also contains the repository), isolates the registry location and appends the eks-anywhere-package-bundles repository to it. Does this help?